package org.powlab.jeye.tests.exception;

public class ExceptionTestFinally22f2 {

    public void test(int x) {
        try {
            return;
        } finally {
            System.out.println("a");
        }
    }
}
